spring-cloud-dataflow-platform-cloudfoundry/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1,5 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
```

Comment:
We should postfix all auto-config classes with `AutoConfiguration` as otherwise it becomes difficult to track down configs. Also we're starting to have a bit of a mess with these classes as they are scattered around different package. We should probably create a new package `spring-cloud-dataflow-autoconfigure` and put everything there.

Reply:
Created https://github.com/spring-cloud/spring-cloud-dataflow/issues/2693  Having different autoconfig classes for different platforms in different packages makes sense to me, or are you referring to something else?